app/assets/javascripts/controllers/ems_common/arbitration_profile_form_controller.js

```diff
@@ -129,5 +129,7 @@ ManageIQ.angular.app.controller('arbitrationProfileFormController', ['$scope', '
     })
   };
 
+  $scope.$watch('_authentication', function() {$scope.arbitrationProfileModel.authentication_id = $scope._authentication.id })
```

Comment:
Aand you may need to check for $scope._authentication being falsy before reading the id here.. :)